src/app/components/LiteSiteSummary/liteSiteTranslations.ts

```diff
@@ -1,11 +1,14 @@
-import { Services } from '#app/models/types/global';
-
-export const defaultTranslations = {
+const defaultTranslations: {
```

Comment:
Why don't we just use the default export here
```suggestion
export default {
```

Reply:
Ahh I remember now, had to revert this, export default doesn't support type annotations

https://github.com/microsoft/TypeScript/issues/13626

Reply:
Ahhhh, thanks for the explanation - I didn't realise.